### PR TITLE
fix: add button font weight

### PR DESCRIPTION
**Pull request by akamkaro**
The button font weight is now correct

### CODE REVIEW

**Comment by oclero** [comment]
A question more for designers: should they add a special case for buttons in typgraphy?

File: src/components/button/Button.module.scss [hunk 1]
```diff
@@ -25,19 +26,22 @@
   &.small {
     padding: 8px 16px;
     height: 32px;
-    @include typography.body-small;
```
Comment (adelego) [inline src/components/button/Button.module.scss:28]: Why did you remove this?

Comment (oclero) [reply src/components/button/Button.module.scss:28]: My guess would be because the buttton does not use the values un `typography.body-small`. Maybe we should have a `typography.button` instead of hardcoded values? (see my other comment)

**Comment by pducolin** [comment]
> A question more for designers: should they add a special case for buttons in typgraphy?

better ask this on figma

File: src/components/button/Button.module.scss (hunk 1 above)
Comment (pducolin) [reply src/components/button/Button.module.scss:28]: Indeed, button doesn't use typography body small - just a subset. I agree it should be a new typography, adding comment on figma to get designer feedback

Comment (akamkaro) [reply src/components/button/Button.module.scss:28]: The font-weight in @mixin body-small was not the good one.
And as it would be only used for the buttons, it is better in my opinion not to add styles in typography.scss, which is a file for shared styles.

Comment (oclero) [reply src/components/button/Button.module.scss:28]: I disagree. I would prefer to not have hardcoded fonts and sizes in the button's code, because if we decide to change, it'll be harder to trace if we have to go into every different component.

**Comment by oclero** [comment]
> > A question more for designers: should they add a special case for buttons in typgraphy?
> 
> better ask this on figma

I've seen you've asked! thanks

File: src/components/button/Button.module.scss (hunk 1 above)
Comment (akamkaro) [reply src/components/button/Button.module.scss:28]: Ok done :)
